tests/test_inclined_exponential.py

```diff
@@ -16,10 +16,15 @@
 #    and/or other materials provided with the distribution.
 #
 
+"""This file contains tests for the InclinedExponential and InclinedSersic classes.
```

Comment:
Unless I missed it, it doesn't seem that there are tests of initializing using ```half_light_radius``` or ```flux_untruncated```.